addons_rksv/pos_rksv/static/src/js/screens.js

```diff
@@ -164,7 +166,8 @@ odoo.define('pos_rksv.screens', function (require) {
      */
     screens.ReceiptScreenWidget.include({
         handle_auto_print: function() {
-           console.log('I WANT TO PRINT LATER')
+            if (this.pos.config.iface_rksv)
```

Comment:
I was not sure about the handle_auto_print function - what does it do here if it is not RKSV related ?